lib/vfscore/mount.c

```diff
@@ -308,11 +308,13 @@ UK_SYSCALL_R_DEFINE(int, umount2, const char*, path, int, flags)
 	return error;
 }
 
+#if UK_LIBC_SYSCALLS
 int
 umount(const char *path)
```

Comment:
Wouldn't make using `UK_SYSCALL_R_DEFINE` instead, more sense? I assume the same for `umount2`

Reply:
`umount` is not actually a syscall. Musl will do:
```
int umount(const char *special)
{
        return syscall(SYS_umount2, special, 0);
}
```
And newlib will use the definition above.

Reply:
You are right, then this is fine.